gen/gen.go

```diff
@@ -26,11 +26,10 @@ import (
 //     └── xgo
 var dirMetadata string
 
-// dirProblem is the location that the test cases should be generated to.
-// This assumes that the generator script lives in the same directory as
-// the problem.
-// Falls back to the present working directory.
-var dirProblem string
+// dirExercise is the location that the test cases should be generated to.
+// This assumes that the generator script lives in the .meta directory within
+// the exercise directory. Falls back to the present working directory.
```

Comment:
ooh, good catch. Left it as is and changed back line 63 in response to your other comment.